.github/workflows/publish.yml

```diff
@@ -40,32 +47,32 @@ jobs:
     needs: run_tests
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Set release notes tag
         run: |
           export RELEASE_TAG_VERSION=${{ github.event.release.tag_name }}
           echo "RELEASE_TAG_VERSION=${RELEASE_TAG_VERSION:1}" >> $GITHUB_ENV
       - name: Set up Python
-        uses: actions/setup-python@v3
+        uses: actions/setup-python@v5
         with:
-          python-version: 3.8
+          python-version: "3.13"
       - name: Install dependencies
         run: make -C cfn-lint-serverless dev
       - name: Upload to PyPi
         run: make -C cfn-lint-serverless release
         env:
           PYPI_USERNAME: __token__
-          PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
+          PYPI_TOKEN: ${{ secrets.PYPI_TOKEN25 }}
```

Comment:
In the future we can configure trust workflows to publish this to pypi.

I'll open an issue.